.github/workflows/zxc-e2e-test.yaml

```diff
@@ -178,6 +178,12 @@ jobs:
         run: |
           ${{ env.CG_EXEC }} npm run ${{ inputs.npm-test-script }}
 
+      - name: RPC relay smoke test
+        if: ${{ runner.os == 'linux' && inputs.npm-test-script == 'test-e2e-relay' && !cancelled() && !failure() }}
+        run: |
+          echo "Skipped smoke test for relay"
+          #.github/workflows/script/relay_smoke_test.sh
```

Comment:
did you leave this commented out intentionally?

Reply:
Yes. 
https://github.com/hashgraph/solo/pull/481#discussion_r1718454106